Signal/src/view controllers/CallViewController.swift

```diff
@@ -472,7 +473,28 @@ class CallViewController: UIViewController, CallDelegate {
         case .answering:
             return NSLocalizedString("IN_CALL_SECURING", comment: "Call setup status label")
         case .connected:
-            return NSLocalizedString("IN_CALL_TALKING", comment: "Call setup status label")
+            if let call = self.call {
+                let callDuration = call.connectionDuration()
+                let callDurationDate = Date(timeIntervalSinceReferenceDate:callDuration)
+                let dateFormatter = DateFormatter()
```

Comment:
Would it be worth caching this DateFormatter? https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/DataFormatting/Articles/dfDateFormatting10_4.html#//apple_ref/doc/uid/TP40002369-SW10